pkg/reconciler/brokercell/controller.go

```diff
@@ -74,6 +77,10 @@ func NewController(
 		logger.Fatal("Failed to create BrokerCell reconciler", zap.Error(err))
 	}
 	impl := v1alpha1brokercell.NewImpl(ctx, r)
+	metricsReporter, err := metrics.NewLatencyReporter()
+	if err != nil {
+		logger.Error("Failed to create latency reporter", zap.Error(err))
+	}
```

Comment:
IMO a user shouldn't really care about this metric. It's more like a metric we want to collect in certain envs (integ tests mostly). I wonder if we need to make a toggle for this metric reporting?

@liu-cong @grantr

Reply:
Sounds like a good idea to disable these by default

Reply:
Makes sense. There are multiple ways to approach that: environment variable, debug mode, ConfigMap, CRD. The first one seems quite suitable here -- just in case, is everyone fine with that?

Reply:
I am fine with an env var

Reply:
Addressed.